### ISSUE TITLE
'got back false' error for hubot rocketchat adapter?

### ISSUE THREAD

**Issue by RichardFoxworthy** [RocketChat/hubot-rocketchat#114]
Trying to spin up a new hubot using

`yo hubot`

but when I say I want to use rocketchat adapter I get this error:

```
[? Bot adapter (campfire) rocketchatgot back false



/usr/local/lib/node_modules/yo/node_modules/rx/dist/rx.js:77
    throw e;
    ^
true
```

Any ideas what is wrong?

**Comment by geekgonecrazy** [comment]
@RichardFoxworthy I've never seen output like that.  I see campfire there.. do you have the campfire installed?

Please ping me in [#support](https://demo.rocket.chat/channel/support) and mention `@aaron.ogle` i'll see if I can get more info.  Hard to troubleshoot this with a big delay between question and response :)

**Comment by RichardFoxworthy** [comment]
Thanks @geekgonecrazy . There seems to be a lot of incidents of the same error reported lately. I got around it by using someone's suggestion to try

`yo hubot --defaults`

**Comment by geekgonecrazy** [comment]
@RichardFoxworthy yeah I apologize.  I'm not sure exactly what's going on.  We're looking into providing some alternative instructions, that hopefully will be a little less problematic.

**Comment by timkinnane** [comment]
For anyone else finding this, the issue is also on the hubot generator:
https://github.com/github/generator-hubot/issues/64

You can get around it by just passing the adapter in the command:
`yo hubot --adapter="rocketchat"`

**Comment by geekgonecrazy** [comment]
@timkinnane ok perfect!  does it ask you any questions at all or does it just spit the bot out?

We're trying to decide how to best update the documentation to make work for the most people.

**Comment by timkinnane** [comment]
It asks all the normal questions, just skips the adapter question. It's just bizarre that defining it as a parameter doesn't throw the exception when giving it as an answer does.

**Comment by nomadic-squirrel** [comment]
I got the same error when using "slack" as an adapter. Using
`yo hubot --adapter="slack"`
Got around it.
